### PR TITLE
Tchalla/hamburger component

### LINKED ISSUE #56: Navbar - Create Hamburger Menu Component
# Requirements

- Create a separate hamburger menu vue component and add it into the `TopNav` vue component.
- Implement a nice UI of your choice for the hamburger menu
- Add a css transition animation for showing and hiding the hamburger menu
- This is just the empty component creation. Links can be added in a separate PR.

----

This has a dependency on PR #54 and #55

### PR DESCRIPTION
**Pull request by Tch4lla**
Resolves #56 

# What changed
Created hamburger component and inserted that into top navbar component. Added hover css to menu options

# Testing instructions
Once screen width is 800 pixels or below, hamburger menu appears. Hover over an option to see css animation

### CODE REVIEW

**Comment by theyokohamalife** [comment]
Hey, @Tch4lla ! Thank you very much for helping out! It looks like there are some linting issues that will need to be fixed before this can be approved. You can run `yarn lint --fix` if you followed the other steps in the README, and it should automatically fix them for you :)

**Comment by Tch4lla** [comment]
@theyokohamalife Thank you kindly. I just ran it, perhaps it's alright now ?

**Review by theyokohamalife — APPROVED** [review]
LGTM. Thank you! 😎

File: components/TopNav.vue [hunk 1]
```diff
@@ -11,30 +11,8 @@
...
-                    src="../assets/images/hamburgerMenu.svg"
-                    alt="hamburger menu" />
-                <div v-if="showMenu">
-                    <div>
-                        <span class="block">Home</span>
-                        <span class="block">About</span>
-                    </div>
-                </div>
-            </div>
+            <HamburgerComponent />
```
Comment (ermish) [inline components/TopNav.vue:14]: @Tch4lla I think a better name would be `HamburgerMenu`. Typically you don't need to include component in the name itself. We can update this in the next PR ✅

**Review by ermish — COMMENTED** [review]
LGTM! 
Left a comment for something we could Fix in the next PR

**Pull request merged**